docs/content/utilities/validation.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: Validation
+description: Utility
+---
+
+import Note from "../../src/components/Note"
+
+This utility provides JSON Schema validation for payloads held within events and response used in AWS Lambda.
+
+**Key features**
+* Validate incoming events and responses
+* Built-in validation for most common events (API Gateway, SNS, SQS, ...)
+* JMESPath support validate only a sub part of the event
+
+## Install
+
+To install this utility, add the following dependency to your project.
+
+```xml
+<dependency>
+    <groupId>software.amazon.lambda</groupId>
+    <artifactId>powertools-validation</artifactId>
+    <version>0.5.0-beta</version>
+</dependency>
+```
+
```

Comment:
I think you should include the additional step for annotations 

https://awslabs.github.io/aws-lambda-powertools-java/utilities/sqs_large_message_handling/

> And configure the aspectj-maven-plugin to compile-time weave (CTW) the aws-lambda-powertools-java aspects into your project. You may already have this plugin in your pom. In that case add the dependency to the aspectLibraries section.